pkg/serverless/trace/propagation/extractor_test.go

```diff
@@ -430,9 +655,6 @@ func TestPropagationStyle(t *testing.T) {
 			expTraceID: w3c.trace.asUint,
 		},
 		{
-			// XXX: This is surprising
-			// The go tracer is designed to always place the tracecontext propagator first
-			// see https://github.com/DataDog/dd-trace-go/blob/6a938b3b4054ce036cc60147ab42a86f743fcdd5/ddtrace/tracer/textmap.go#L231
```

Comment:
It is no longer surprising cuz the issue was fixed a while back. This now works as expected.